src/main/resources/assets/pipez/lang/de_de.json

```diff
@@ -0,0 +1,85 @@
+{
+  "itemGroup.pipez": "Röhrez",
+  "block.pipez.item_pipe": "Item Röhre",
+  "block.pipez.fluid_pipe": "Flüssigkeits Röhre",
+  "block.pipez.energy_pipe": "Energie Röhre",
+  "block.pipez.universal_pipe": "Universele Röhre",
+  "block.pipez.gas_pipe": "Gas Röhre",
+  "item.pipez.basic_upgrade": "Normale Röhre Verbesserung",
+  "item.pipez.improved_upgrade": "Verbesserte Röhre Verbesserung",
+  "item.pipez.advanced_upgrade": "Erweiterte Röhre Verbesserung",
+  "item.pipez.ultimate_upgrade": "Ultimative Röhre Verbesserung",
+  "item.pipez.infinity_upgrade": "Unendliche Röhre Verbesserung",
+  "item.pipez.wrench": "Röhren Zange",
+  "item.pipez.filter_destination_tool": "Filter Ziel Gerät",
+  "tooltip.pipez.wrench": "Klicke auf eine Röhre um sie einzustellen",
```

Comment:
"Klicke auf eine Röhre, um sie einzustellen"

Reply:
How would you change it?